### PR TITLE
refactor(console + mapi): remove old theme code

### PR DESCRIPTION
**Pull request by jourdiw**
## Issue

https://gravitee.atlassian.net/browse/APIM-3061

## Description

Remove console theme code from the front and back.

There should be no regressions.

## Additional context

🏗️ Your changes can be tested here and will be available soon:
      Console: [https://pr.team-apim.gravitee.dev/5634/console](https://pr.team-apim.gravitee.dev/5634/console)
      Portal: [https://pr.team-apim.gravitee.dev/5634/portal](https://pr.team-apim.gravitee.dev/5634/portal)
      Management-api: [https://pr.team-apim.gravitee.dev/5634/api/management](https://pr.team-apim.gravitee.dev/5634/api/management)
      Gateway v4: [https://pr.team-apim.gravitee.dev/5634](https://pr.team-apim.gravitee.dev/5634)
      Gateway v3: [https://pr.gateway-v3.team-apim.gravitee.dev/5634](https://pr.gateway-v3.team-apim.gravitee.dev/5634)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-cuzttuohya.chromatic.com)

### CODE REVIEW

**Comment by jourdiw** [comment]
@loriepisicchio @gaetanmaisse What about applying this to 4.0 + 4.1 ?

**Review by ThibaudAV — COMMENTED** [review]
haha maybe it's not juste a refactor but a BC too 🙈

File: gravitee-apim-console-webui/src/management/configuration/theme/theme.component.ts [hunk 1]
```diff
@@ -1,24 +0,0 @@
...
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-const ThemeComponent: ng.IComponentOptions = {
-  bindings: {
-    views: '<',
-  },
-  controller: 'ThemeController',
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/management/configuration/theme/theme.component.ts:20]: This is the theme for the portal ? 🤔

Comment (jourdiw) [reply gravitee-apim-console-webui/src/management/configuration/theme/theme.component.ts:20]: It is 🙃 I have FP and have saved the portal related assets and have renamed them. It still builds and runs no error 😅

**Comment by loriepisicchio** [comment]
I agree with @ThibaudAV the commits should not only be `refactor` but probably indicate a breaking change. 

Also to answer your question, I would not apply on earlier version IMHO (I don't think it's worth taking the risk)

**Review by loriepisicchio — APPROVED** [review]

**Comment by jourdiw** [comment]
After discussion with @loriepisicchio, @ThibaudAV, and @gaetanmaisse, this will stay on master and not be merged onto 4.0 + 4.1.

File: gravitee-apim-console-webui/src/components/theme/theme-element.directive.ts [hunk 2]
```diff
@@ -1,39 +0,0 @@
...
-        style = style[value];
-      });
-
-      elem.css(style);
-    }
-  },
-  controller: ThemeElementController,
-};
-
-export default ThemeElementDirective;
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/components/theme/theme-element.directive.ts:39]: You should also remove all use of this directive : find : `gv-Theme-Element`

**Review by ThibaudAV — APPROVED** [review]
The code looks good, but I haven't tested the application yet.

File: gravitee-apim-console-webui/src/components/theme/theme-element.directive.ts (hunk 2 above)
Comment (jourdiw) [reply gravitee-apim-console-webui/src/components/theme/theme-element.directive.ts:39]: Done.

**Pull request merged**